packages/react-composites/src/composites/common/SendBox.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React, { useMemo } from 'react';
+import { SendBoxStylesProps, SendBox as SimpleSendBox } from '@internal/react-components';
+import { usePropsFor } from '../ChatComposite/hooks/usePropsFor';
+import { ChatCompositeOptions } from '../ChatComposite';
+/* @conditional-compile-remove(file-sharing) */
+import { ChatAdapter } from '../ChatComposite';
+/* @conditional-compile-remove(file-sharing) */
+import { fileUploadsSelector } from '../ChatComposite/selectors/fileUploadsSelector';
+/* @conditional-compile-remove(file-sharing) */
+import { useSelector } from '../ChatComposite/hooks/useSelector';
+
+/* @conditional-compile-remove(rich-text-editor) */
+const RichTextSendBoxWrapper = React.lazy(() =>
+  import('./RichTextSendBoxWrapper').then((module) => ({ default: module.RichTextSendBoxWrapper }))
+);
+
+/**
+ * @private
+ */
+export type SendBoxProps = {
+  options?: ChatCompositeOptions;
+  styles?: SendBoxStylesProps;
+  /* @conditional-compile-remove(file-sharing) */
+  adapter: ChatAdapter;
+};
+
+/**
+ * @private
+ */
+export const SendBox = (props: SendBoxProps): JSX.Element => {
+  const {
+    options,
+    styles,
+    /* @conditional-compile-remove(file-sharing) */
+    adapter
+  } = props;
+
+  const sendBoxProps = usePropsFor(SimpleSendBox);
+
+  /* @conditional-compile-remove(file-sharing) */
+  const activeFileUploads = useSelector(fileUploadsSelector).files;
+
+  const sendBoxStyles = useMemo(() => {
+    return Object.assign({}, styles);
+  }, [styles]);
+
+  /* @conditional-compile-remove(rich-text-editor) */
+  if (options?.richTextEditor !== false) {
+    return <RichTextSendBoxWrapper />;
```

Comment:
Do we need to use `Suspense` with `fallback` here?

Reply:
Right, still to add that, I'll do that now.